scripts/open_url_in_instance.sh

```diff
@@ -2,7 +2,7 @@
 # initial idea: Florian Bruhin (The-Compiler)
 # author: Thore Bödecker (foxxx0)
 
-_url="$1"
+_url="${1//	/ }"
```

Comment:
> ./scripts/open_url_in_instance.sh: 5: ./scripts/open_url_in_instance.sh: Bad substitution

could you do this in a more portable way, if this is needed at all?

Reply:
I see the shebang has been changed to `/bin/sh`.  😏  Just a second...